thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/formatter/channels/JiraContentFormatter.java

```diff
@@ -156,13 +135,43 @@ private JiraEntity buildJiraEntity(String jiraTemplate, Map<String, Object> temp
       Template template = freemarkerConfig.getTemplate(jiraTemplate);
       template.process(templateValues, out);
 
-      String alertEmailHtml = new String(baos.toByteArray(), CHARSET);
+      description = new String(baos.toByteArray(), CHARSET);
+    } catch (Exception e) {
+      description = "Found an exception while constructing the description content. Pls report & reach out"
+          + " to the Thirdeye team. Exception = " + e.getMessage();
+    }
+
+    return description;
```

Comment:
When an exception is thrown, should `description` be `null` or empty, as I see that you assign empty string for snapshot if there is an error.

Reply:
When an exception is thrown in this method, the description is neither null or empty. It will store and print an error message in the jira ticket.

Reg. Snapshot- this is not enabled right now. I will test and enable this in a subsequent PR.